config/storage/backend/quota/quota-file.json

```
@@ -28,10 +28,22 @@
 
     {
       "comment": "Simple wrapper for another DataAccessor but adds validation",
-      "@id": "urn:solid-server:default:FileDataAccessor",
+      "@id": "urn:solid-server:default:ValidatingFileDataAccessor",
       "@type": "ValidatingDataAccessor",
       "accessor": { "@id": "urn:solid-server:default:AtomicFileDataAccessor" },
       "validator": { "@id": "urn:solid-server:default:QuotaValidator" }
+    },
+    {
+      "comment": "Simple wrapper for another DataAccessor but removes specific metadata entries",
```

```suggestion
      "comment": "Removes content-length metadata",
```
Comments in the configs should describe what this specific instance does.